src/ConstructiveSolidGeometry/plotting/CSG/CSG.jl

```diff
@@ -0,0 +1,38 @@
+@recipe function f(csg::ConstructiveSolidGeometry.CSGUnion)
+    linecolor --> :black
+    csguniontype --> :union
+    @series begin 
+        linestyle := :solid
+        csg.a
+    end
+    @series begin 
+        linestyle := :solid
+        csg.b
+    end
+end
+@recipe function f(csg::ConstructiveSolidGeometry.CSGDifference)
+    linecolor --> :black
+    @series begin
+        linestyle := :solid
+        csguniontype --> :difference
+        csg.a
+    end
+    @series begin
+        linestyle := :dash
+        csg.b
+    end
+end
```

Comment:
Ẁhat happens to `differences` of `differences`?

Reply:
Should be parsed recursively I think.
So a `CSGUnion` as `csg.b` of a `CSGDifference` should be plotted with dashed lines.

Reply:
And a union of differences?
Looks like the `:=` syntax ignores all `linestyles` that might come afterward.